feature-libs/smartedit/ng-package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "../../dist/smartedit",
+  "lib": {
+    "entryFile": "./public_api.ts",
+    "umdModuleIds": {
+      "@spartacus/core": "core",
+      "rxjs": "rxjs"
```

Comment:
I would expect that `webApplicationInjector.js` file would be handled as an asset somewhere here (https://github.com/ng-packagr/ng-packagr/blob/master/docs/copy-assets.md)?